runtime/vm/linearswalk.c

```diff
@@ -396,16 +396,20 @@ lswRecordSlot(J9StackWalkState * walkState, const void * slotAddress, UDATA slot
 
 
 static char *
-lswDescribeObject(J9VMThread * vmThread, char * dataPtr, J9Class ** class, J9Object * object)
+lswDescribeObject(J9VMThread * vmThread, char * dataPtr, UDATA dataSize, J9Class ** class, J9Object * object)
```

Comment:
Suggest `size_t` is preferable to `UDATA` here and in the signature of `lswDescribeSlot()`.